apps/www/components/Audio/AudioPlayer/ui/AudioCover.tsx

```diff
@@ -69,3 +68,29 @@ const AudioCover = ({
 }
 
 export default AudioCover
+
+/**
+ * Get a resized and cropped image url
+ * @param imageUrl image to crop
+ * @param size defines the width and height of the image
+ * @param audioCoverCrop crop parameters
+ * @returns image url with crop and resize parameters
+ */
+export function getImageCropURL(
```

Comment:
Put this in a separate file? I don't think it's great to export random functions from component files (too easy to create circular dependencies).